website/content/docs/api-clients/api/pagination.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 layout: docs
 page_title: API list pagination
-description: Learn how the API pagination and cache works in Boundary to prevent system resources from being overwhlemed and to help you find resources.
+description: >-
+  Learn how the API pagination and cache works in Boundary to prevent system resources from being overwhlemed and to help you find resources.
```

Comment:
```suggestion
  Learn about API pagination and how the cache works in Boundary to prevent system resources from being overwhelmed and to help you find resources.
```